ext/Cargo.toml

```diff
@@ -1,18 +1,23 @@
 [package]
 name = "ext"
 version = "0.1.0"
-authors = ["Hood Chatham <hood@mit.edu>", "Dexter Chua <dexter@math.harvard.edu>"]
+authors = [
+    "Hood Chatham <hood@mit.edu>",
+    "Dexter Chua <dexter@math.harvard.edu>",
+]
```

Comment:
Maybe add yourself here?

Reply:
Ah yes, some toml changes are due to my autoformatter. Thanks for the suggestion, I'll add my name